debian/rules

```diff
@@ -15,6 +15,7 @@ override_dh_auto_test:
 override_dh_auto_build:
 override_dh_auto_install:
 	make
+	mkdir -p /var/lib/crowdsec/cache/
```

Comment:
I think you want `debian/crowdsec-cloudflare-bouncer/var/lib/crowdsec/cache/` as a target,